installer/PowerToysBootstrapper/bootstrapper/Resources.resx

```diff
@@ -153,5 +153,12 @@
   </data>
   <data name="SNOOZE_BUTTON" xml:space="preserve">
     <value>Snooze</value>
+  </data>
+  <data name="GITHUB_NEW_VERSION_USING_LOCAL_BUILD_ERROR" xml:space="preserve">
+    <value>Cannot update from a local build!</value>
+    <comment>User cannot autoupdate from a locally-built PowerToys version</comment>
+  </data>
+  <data name="GITHUB_NEW_VERSION_CHECK_ERROR" xml:space="preserve">
+    <value>There was a problem while checking for new version.</value>
```

Comment:
```suggestion
    <value>Failed to connect to the server. Check your network connection or retry later.</value>
```

Reply:
This is supposed to be a generic error message. The network-related error text is appended to this message.

Reply:
What are the possible cases?

Reply:
Anything network-related, e.g. timeout, malformed json response from github, github returning 503 HTTP status code etc.